### PR TITLE
fix(inputMsg): input can be also a commit message

### PR DESCRIPTION
**Pull request by stevemao**
Fix regression from https://github.com/kentcdodds/validate-commit-msg/commit/d850cf47265d5ff18e86a3a743678ecfc7d66c03

### CODE REVIEW

**Comment by stevemao** [comment]
CC @krishna63

File: lib/cli.js [hunk 1]
```diff
@@ -18,11 +18,13 @@ var getGitFolder = require('./getGitFolder');
 // hacky start if not run by mocha :-D
 // istanbul ignore next
 if (process.argv.join('').indexOf('mocha') === -1) {
-  var commitMsgFileOrText,
+  var arg = process.argv[2],
+      isInputFileName = arg && fs.existsSync(gitFolder  + arg),
       gitFolder = getGitFolder() + '/',
```
Comment (krishna63) [inline lib/cli.js]: @stevemao : can you declare gitfolder variable before using it

**Comment by krishna63** [comment]
@stevemao : May i know the reason behind this PR , PLS

**Comment by stevemao** [comment]
Specifying commit message as input doesn't work any more

File: lib/cli.js (hunk 1 above)
Comment (stevemao) [inline lib/cli.js]: yup sorry...

**Comment by stevemao** [comment]
I'm closing this since https://github.com/conventional-changelog/validate-commit-msg/pull/92#issuecomment-313691307 is merged. Please comment if I'm wrong @maxcnunes @Garbee Thanks.

**Pull request closed without merging**